corelib-schemaorg/src/main/java/eu/europeana/corelib/edm/model/schemaorg/Timespan.java

```diff
@@ -0,0 +1,69 @@
+package eu.europeana.corelib.edm.model.schemaorg;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import ioinformarics.oss.jackson.module.jsonld.annotation.JsonldType;
+
+import java.util.List;
+
+@JsonldType(SchemaOrgConstants.TYPE_TIMESPAN)
+@JsonInclude(value = JsonInclude.Include.NON_EMPTY)
+@JsonPropertyOrder({"@type", "@id", 
+    SchemaOrgConstants.PROPERTY_NAME,SchemaOrgConstants.PROPERTY_ALTERNATE_NAME, SchemaOrgConstants.PROPERTY_DESCRIPTION,
+	SchemaOrgConstants.PROPERTY_BEGIN_DATE, SchemaOrgConstants.PROPERTY_END_DATE,SchemaOrgConstants.PROPERTY_SAME_AS,
+	SchemaOrgConstants.PROPERTY_IS_PART_OF, SchemaOrgConstants.PROPERTY_HAS_PART, SchemaOrgConstants.PROPERTY_NEXT_IN_SEQUENCE})
+public class Timespan extends ContextualEntity {

```

Comment:
We decided to map Timespans to the Thing class only. See mapping file (starting with row 108 https://docs.google.com/spreadsheets/d/1E2P0kgdJ08lqLdWKZK_3fNeiXOuuCcgzarjlSYtEygc/edit#gid=198127005 ). This class is not needed, please remove it.

Reply:
I think the class is still needed in case someone wants to modify the serialization in the future (e.g. add new fields). It does not cause any problems to the remaining corelib code.

Reply:
This class will be created when the specifications are updated and the class is required. For the time being it is not the case.

Reply:
there is no Timespan type in schema.org, the decision from specifications was to use the Thing class https://schema.org/docs/search_results.html?q=thing

Reply:
I have added the Timespan class as agreed with no fields in addition to the Thing class.